src/components/asset-list/RecyclerAssetList2/profile-header/ProfileActionButtonsRow.tsx

```diff
@@ -336,8 +337,14 @@ function MoreButton() {
     [handlePressConnectedApps, handlePressCopy, handlePressQRCode]
   );
 
+  const onMenuWillShow = React.useCallback(() => {
+    // update state to potentially hide the menu button
+    setActiveWCV2Sessions(getAllActiveSessionsSync());
+  }, [setActiveWCV2Sessions]);
```

Comment:
Instead of watching for updates in the background, this lazily checks for sessions (no network req) every time it opens, which then shows the link in the overflow menu.